tests/unit/test_start_handlers.py

```diff
@@ -1255,3 +1199,39 @@ async def test_start_happy_path_cy(self):
                 continue
             # outputs failed key as msg
             self.assertEqual(eq_payload[key], token[key], key)
+
+    @unittest_run_loop
+    async def test_post_start_for_receiptReceived_true_ew_e(self):
+        uac_json = self.uac_json_e.copy()
+        uac_json['receiptReceived'] = True
+        with self.assertLogs('respondent-home', 'WARNING') as cm, aioresponses(
+                passthrough=[str(self.server._root)]) as mocked:
+            mocked.get(self.rhsvc_url, payload=uac_json)
+
+            response = await self.client.request('POST',
+                                                 self.post_start_en,
+                                                 data=self.start_data_valid)
+
+            self.assertLogEvent(cm, "attempt to use an inactive access code")
+            self.assertEqual(response.status, 200)
+            contents = str(await response.content.read())
+            self.assertSiteLogo('en', contents)
+            self.assertIn(self.content_start_uac_expired_en, contents)
```

Comment:
Should probably rename this (and the welsh one) as the template is not called 'expired' any more

Reply:
This has been renamed